src/upgradeDB.js

```
@@ -295,6 +296,46 @@
+            { name: roleName },
+            roleData,
+            { upsert: true, new: true }
+          );
+          logger.info(`Role ${roleName} created or updated`);
+        }
+
+        const users = await UserModel.find();
+
+        const userPromises = users.map(async (user) => {
+          let newGroup = 'manager';
+          if ((user.groups && user.groups.includes('admin')) || user.superUser) {
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)